docs/specs/#2072 - Add some PowerToys utilities/spec.md

```diff
@@ -0,0 +1,165 @@
+---
+author: Kayla Cinnamon @cinnamon-msft
+created on: 2023-12-22
+last updated: 2024-01-04
+issue id: #2072
+---
+
+# Add some PowerToys utilities
+
+## 1. Overview
+
+### 1.1 Establish the Problem
+
+Dev Home is intended to be a centralized location for developers to help them stay productive on Windows. One feature we had been thinking of is adding helpful utilities into Dev Home for quick and easy access.
+
+### 1.2 Introduce the Solution
+
+PowerToys has created a few utilities with the original intent of migrating them into Dev Home. These utilities include Registry Preview, Hosts File Editor, and Environment Variables. We can add these utilities into Dev Home as the "stable" versions of these tools, while maintaining their "preview" versions in PowerToys. By adding these utilities to Dev Home, we can provide developers with more value out of the box in Windows while also provide more awareness to PowerToys.
+
+### 1.3 Rough-in Designs
```

Comment:
```suggestion
### 1.3 Rough-in Designs for initial release
```

Want to clearly state for the TLDR readers this is a first iteration, not long term